### PR TITLE
Implement node publish secret mount policy

### PR DESCRIPTION
**Pull request by TrafalgarZZZ**
### Ⅰ. Describe what this PR does
Implement node publish secret mount policy

### Ⅱ. Does this pull request fix one issue?

NONE

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: api/v1alpha1/thinruntimeprofile_types.go [hunk 1]
```diff
@@ -21,8 +21,13 @@ import (
...
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+type NodePublishSecretPolicy string
+
+const (
+	NotMountNodePublishSecret        NodePublishSecretPolicy = "NotMountNodePublishSecret"
+	MountNodePublishSecretIfExists   NodePublishSecretPolicy = "MountNodePublishSecretIfExists"
+	CopyNodePublishSecretIfNotExists NodePublishSecretPolicy = "CopyNodePublishSecretIfNotExists"
```
Comment (cheyang) [inline api/v1alpha1/thinruntimeprofile_types.go:29]: How about renaming `CopyNodePublishSecretIfNotExists` to `CopyToMountNodePublishSecretIfNotExists` ?

Comment (TrafalgarZZZ) [reply api/v1alpha1/thinruntimeprofile_types.go]: Done. Renamed it with `CopyNodePublishSecretAndMountIfnotExists`

File: api/v1alpha1/thinruntimeprofile_types.go [hunk 2]
```diff
@@ -39,6 +44,10 @@ type ThinRuntimeProfileSpec struct {
 	// Volumes is the list of Kubernetes volumes that can be mounted by runtime components and/or fuses.
 	// +optional
 	Volumes []corev1.Volume `json:"volumes,omitempty"`
+
+	// NodePublishSecretPolicy describes the policy to decide which to do with node publish secret when mounting an existing persistent volume.
+	// +kubebuilder:default=MountNodePublishSecretIfExists
+	NodePublishSecretPolicy NodePublishSecretPolicy `json:"nodePublishSecretPolicy,omitempty"`
```
[the remark this answers is not in the document]
Comment (TrafalgarZZZ) [reply api/v1alpha1/thinruntimeprofile_types.go:51]: Done.

**Review by cheyang — APPROVED** [review]

**Pull request merged**